conf.d/mongo.yaml.example

```diff
@@ -13,6 +13,10 @@ instances:
 
     # Optional SSL parameters, see https://github.com/mongodb/mongo-python-driver/blob/2.6.3/pymongo/mongo_client.py#L193-L212
     # for more details
+    # For `ssl_ca_certs` should be:
```

Comment:
Small error probably: shouldn't it be `ssl_cert_reqs` ? https://github.com/mongodb/mongo-python-driver/blob/2.6.3/pymongo/mongo_client.py#L201-L208

Could you also put this under the `ssl_cert_reqs` explanation ?
I.e
```
# ssl_cert_reqs: # Specifies whether a certificate is required from the other side of the connection, 
# and whether it will be validated if provided.
# Possible values:
#   * 0 for ssl.CERT_NONE (certificates ignored)
#   * 1 for ssl.CERT_OPTIONAL (not required, but validated if provided)
#   * 2 for ssl.CERT_REQUIRED (required and validated)